bounce/db/membership.py

```diff
@@ -1,10 +1,14 @@
 """Defines the schema for the Memberships table in our DB."""
 
-from sqlalchemy import Column, ForeignKey, Integer, func
+from sqlalchemy import Column, ForeignKey, Integer, String, func
+from sqlalchemy.dialects.postgresql import ENUM
 from sqlalchemy.orm import relationship
 from sqlalchemy.types import TIMESTAMP
 
-from . import BASE
+from . import BASE, PermissionError
+
+# Defining a enum type for role allocation
+ROLE = ENUM('President', 'Admin', 'Member', name='role')
```

Comment:
Since you'll need this role in multiple files under the `db` module, I'd recommend putting it in `db/__init__.py` and importing it where you need it.

Reply:
Thanks, I have put it in the __init__.py file.